models/cryptopunks/ethereum/cryptopunks_ethereum_punk_transfers.sql

```diff
@@ -20,36 +21,36 @@ from
                 , punk_id
                 , evt_tx_hash
         from
-        (   select  a.from
+        (   select  a."from"
                 , a.to
                 , a.evt_block_time
                 , date_trunc('week',a.evt_block_time) as evt_block_time_week
                 , a.evt_block_number
                 , a.evt_index
-                , case when a.evt_tx_hash = lower('0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e') and a.evt_index = 11 then 675
-                        when a.evt_tx_hash = lower('0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e') and a.evt_index = 27 then 675
-                        when a.evt_tx_hash = lower('0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e') and a.evt_index = 25 then 2138
-                        when topic1 = '0x05af636b70da6819000c49f85b21fa82081c632069bb626f30932034099107d8' then cast(bytea2numeric_v3(substring(data from 3)) as int)
-                        else cast(bytea2numeric_v3(substring(topic2 from 3)) as int) end as punk_id
+                , case when a.evt_tx_hash = 0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e and a.evt_index = 11 then UINT256 '675'
+                        when a.evt_tx_hash = 0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e and a.evt_index = 27 then UINT256 '675'
+                        when a.evt_tx_hash = 0x76d32b465ca332bbbe74f7a1834c6d354125f6950168c6123f8ab07440bc285e and a.evt_index = 25 then UINT256 '2138'
+                        when topic1 = 0x05af636b70da6819000c49f85b21fa82081c632069bb626f30932034099107d8 then cast(bytearray_to_integer(data) as UINT256)
```

Comment:
So i crossed checked trion and spark, and `bytearray_to_uint256(data)` and `bytearray_to_uint256(topic2)` seem to be returning correct numbers. 

In your example, if you select * instead of selecting bytearray..., the result is still empty. I dug around for hours and couldn't find the issue. I think the issue is that we can find 
````
topic1 in   ( 0x58e5d5a525e3b40bc15abaa38b5882678db1ee68befd2f60bafe3a7fd06db9e3 -- PunkBought
                                , 0x05af636b70da6819000c49f85b21fa82081c632069bb626f30932034099107d8 -- PunkTransfer
                                )
```
in ethereum.logs.

All the other tables models seem to be returning recent data, but not `cryptopunks_ethereum_punk_transfers`.

Reply:
`cryptopunks_ethereum_punk_transfers` only contains data from `source('cryptopunks_ethereum','CryptoPunksMarket_evt_Assign')` which only goes to 2017

Reply:
Ah ok bizarre so maybe it's something they implemented but was never actually used 🤔

Reply:
Not a blocker I think!

Reply:
Sorry forgot to mention that the spark version contains all the recent transactions: https://dune.com/queries/3051399

Reply:
@0xRobin Not sure if you saw this PR. Do you have any idea what could be happening here?

Reply:
I'll take a look

Reply:
@couralex6 `topic1` -> `topic0` in DuneSQL.